libraries/core-react/src/Popover/Popover.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { forwardRef, useRef, HTMLAttributes, ReactNode } from 'react'
+import styled from 'styled-components'
+import { PopoverItem } from './PopoverItem'
+
+const Container = styled.div`
+  position: relative;
+  display: inline-flex;
+  justify-content: center;
+`
+
+const Anchor = styled.div`
+  &:focus {
+    outline: none;
+  }
+`
+type PopoverChild = {
+  type?: { displayName?: string }
+} & ReactNode
+
+export type Props = {
+  /* Popover placement relative to anchor */
+  placement?:
+    | 'topLeft'
+    | 'top'
+    | 'topRight'
+    | 'rightTop'
+    | 'right'
+    | 'rightBottom'
+    | 'bottomLeft'
+    | 'bottom'
+    | 'bottomRight'
+    | 'leftTop'
+    | 'left'
+    | 'leftBottom'
+  /**  On Close function */
+  onClose?: () => void
+  /**  Open activates <PopoverItem/> */
+  open?: boolean
+} & HTMLAttributes<HTMLDivElement>
```

Comment:
If `children` is required in the component, don't we have to explicitly type it? My guess is that `& HTMLAttributes<HTMLDivElement>` only includes `children` as optional? 🤔

Reply:
I did as we talked about at today's standup, replicated React's native functionality with allowing no childrens. I also added tests to highlight.